spec/Block-Synchronization-Extension.md

```diff
@@ -53,32 +53,17 @@ Request corresponding bodies for each hash.
 * Restriction:
   * MUST include at least one item
 
-
-### GetStateHead
-
-```
-GetStateHead(block_hash)
-```
-
-Request corresponding state head for block of `block_hash`.
-
-* Identifier: 0x06
-* Restriction: Block number of requested block MUST be multiple of 214.
-
-
 ### GetStateChunk
 
 ```
-GetStateChunk(block_hash, tree_root)
+GetStateChunk(block_hash, chunk_root)
```

Comment:
See the proposal closely. It is a list of chunk roots.
https://research.codechain.io/t/snapshot-sync-proposal/21

Reply:
Sorry for the mistake.
I think this format is better though. I left a suggestion to the proposal.